src/keboola_mcp_server/tools/components/model.py

```diff
@@ -1,354 +1,550 @@
     """
-    A Component containing detailed information about the Keboola Component, including its capabilities,
-    documentation, and configuration schemas.
+    Complete component representation with full details.
+
+    Contains comprehensive component information including documentation,
+    configuration schemas, and metadata. Used by get tools where detailed
+    component information is needed.
     """
 
+    # Core component metadata (shared with ComponentSummary)
+    component_id: str = Field(
+        description='Component ID',
+    )
+    component_name: str = Field(
+        description='Component name',
+    )
+    component_type: str = Field(
+        description='Component type',
+    )
     component_categories: list[str] = Field(
         default_factory=list,
-        description='The categories the component belongs to.',
-        validation_alias=AliasChoices(
-            'componentCategories', 'component_categories', 'component-categories', 'categories'
-        ),
-        serialization_alias='categories',
+        description='Component categories',
     )
-    documentation_url: Optional[str] = Field(
+    capabilities: ComponentCapabilities = Field(description='Component capabilities')
+
+    # Detailed metadata (not in ComponenSummary)
+    documentation_url: str | None = Field(
         default=None,
-        description='The url where the documentation can be found.',
-        validation_alias=AliasChoices('documentationUrl', 'documentation_url', 'documentation-url'),
-        serialization_alias='documentationUrl',
+        description='URL to component documentation',
     )
-    documentation: Optional[str] = Field(
+    documentation: str | None = Field(
         default=None,
-        description='The documentation of the component.',
-        serialization_alias='documentation',
+        description='Component documentation text',
     )
-    configuration_schema: Optional[dict[str, Any]] = Field(
+    configuration_schema: dict[str, Any] | None = Field(
         default=None,
-        description='The configuration schema for the component.',
-        validation_alias=AliasChoices('configurationSchema', 'configuration_schema', 'configuration-schema'),
-        serialization_alias='configurationSchema',
+        description='JSON schema for root configuration validation',
     )
-    configuration_row_schema: Optional[dict[str, Any]] = Field(
+    configuration_row_schema: dict[str, Any] | None = Field(
         default=None,
-        description='The configuration row schema of the component.',
-        validation_alias=AliasChoices('configurationRowSchema', 'configuration_row_schema', 'configuration-row-schema'),
-        serialization_alias='configurationRowSchema',
+        description='JSON schema for row configuration validation',
     )
 
+    # MCP-specific metadata
+    links: list[Link] = Field(default_factory=list, description='MCP-specific links for UI navigation')
```

Comment:
```suggestion
    links: list[Link] = Field(default_factory=list, description='Links for UI navigation')
```
I wouldn't say 'MCP-specific': 1) this whole class is MCP-specific in a sense, since it's an output of an MCP tool 2) links seem like a pretty 'generic' attribute to have